src/AppInstallerCommonCore/Manifest/ManifestValidation.cpp

```diff
@@ -71,13 +71,15 @@ namespace AppInstaller::Manifest
                 { AppInstaller::Manifest::ManifestError::SchemaHeaderManifestTypeMismatch , "The manifest type in the schema header does not match the ManifestType property value in the manifest."sv },
                 { AppInstaller::Manifest::ManifestError::SchemaHeaderManifestVersionMismatch, "The manifest version in the schema header does not match the ManifestVersion property value in the manifest."sv },
                 { AppInstaller::Manifest::ManifestError::SchemaHeaderUrlPatternMismatch, "The schema header URL does not match the expected pattern."sv },
+                { AppInstaller::Manifest::ManifestError::InvalidPortableFiletype, "The file type of the referenced file is not allowed."sv },
             };
 
             return ErrorIdToMessageMap;
         }
     }
-    std::vector<ValidationError> ValidateManifest(const Manifest& manifest, bool fullValidation)
+    std::vector<ValidationError> ValidateManifest(const Manifest& manifest, ManifestValidateOption& validateOption)

```

Comment:
Can you revert all these changes back to (and update the callers)
 std::vector<ValidationError> ValidateManifest(const Manifest& manifest, bool fullValidation)
 
I feel the ValidationOptions is supposed to be processed at top level, this is the actual internal operation.